tests/test_icecream.py

```diff
@@ -464,6 +470,7 @@ def testSingleTupleArgument(self):
         self.assertEqual(pair, ('(a, b)', '(1, 2)'))
 
     def testMultilineContainerArgs(self):
+        ic.lineWrapWidth = icecream.DEFAULT_LINE_WRAP_WIDTH
```

Comment:
Is this still needed now you have it in setUp?

Reply:
Nope! Will remove. Good spot.